source/docs/articles/sites/using-tmp-files.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Temporary File Management with Multiple Application Containers
+description: Instructions for using temporary files in distributed environments.
+category:
+- getting-started
+keywords: tmp, temp files, tmp files, temporary files, mulitiple application containers, distributed environments
+---
+Live sites on Professional plans and above have multiple [Application Containers](/docs/articles/sites/all-about-application-containers).  Pantheon's distributed system means that requests are spread between all of the available application servers, which is part of how we help the site scale. However, the `/tmp` directory on one instance is not able to access the `/tmp` resources on another application server. The solution is to create a common, or shared, directory within the file mount which is accessible by all environments.
+
+## Create a Common Directory
+
+Create a `tmp/` directory within the network file mount, which is common across all application containers. Use the following code when applicable to write temporary resources to shared directories:
+
+**Drupal**: `public://sites/default/files/tmp`  
+**WordPress**: `public://wp-content/uploads`
+
+This will allow for consistent execution of requests to temporary files for sites with more than one application container.
+
+## Private Temporary Files
+To manage your temporary files privately and reliably for when multiple application containers are required to process the request, add this to your code: `private://tmp`
```

Comment:
So far I have not been able to change the default `/tmp` dir used by PHP. 

I added the following to wp-config.php but it did not work:
```
define( 'WP_TEMP_DIR' , '/wp-content/uploads/') ;
```

I added `<?php echo sys_get_temp_dir(); ?>` to `header.php` to display the tmp path.

Maybe CSE can help test/verify solutions for mapping an alternate tmp dir for WP users? I'm not having much luck.